compose/docker-compose.arm.yml

```diff
@@ -0,0 +1,12 @@
+# Mark Shust's Docker Configuration for Magento
+# (https://github.com/markshust/docker-magento)
+#
+# Include this configuration file for ARM environments.
+#
+# Version 38.0.0
+
+version: "3"
+
+services:
+  elasticsearch:
+    image: markoshust/magento-elasticsearch:7.9
```

Comment:
This file isn't _really_ necessary, but is present to demonstrate that ARM machines _need_ `7.9` to get a native image. Versions `<7.8` do not provide multi-platform images.